hack/verify-kep-structure.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Check that the required section headers (from the kep template) are
+# present in the kep.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TEMPLATE=$(dirname $0)/../keps/NNNN-kep-template/README.md
+
+MODIFIED_KEPS=$(git log --name-only --pretty= "${PULL_BASE_SHA:-origin/master}.." -- |
+    (grep '^keps.*\.md$' || true) |
+    sort -u)
+RC=0
+
+for KEP in $MODIFIED_KEPS; do
+    echo "Checking ${KEP}"
+
+    # Check for title which starts with #
+    if ! grep -q '^# ' $KEP; then
+        echo "$KEP is missing a title"
```

Comment:
nit: I like writing errors to STDERR
```suggestion
        echo "$KEP is missing a title" >&2
```